.github/workflows/stale.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies, run tests, and report the coverage with a variety of Python versions and OSs
```

Comment:
Wrong doc. Please fix

Reply:
I have fixed it. I will be careful about it next time. Thanks.